stix_shifter_modules/nozomi_vantage/configuration/config.json

```diff
@@ -21,6 +21,15 @@
     "selfSignedCert": {
       "type": "password",
       "optional": true
+    },
+    "options": {
```

Comment:
The minimum page_size should probably be 100. Users shouldn't really need a page_size that small.

Reply:
Updated as per the suggestion.